<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>589</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/png/bg3.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="watericon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>251</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/png/water_20.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="1">
        <widget class="QLabel" name="label_ip_addr">
         <property name="styleSheet">
          <string notr="true">font: 500 12pt &quot;文泉驿等宽正黑&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">font: 500 10pt &quot;文泉驿等宽正黑&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>TCP端口</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_tcp_port">
         <property name="styleSheet">
          <string notr="true">font: 500 12pt &quot;文泉驿等宽正黑&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">font: 500 10pt &quot;文泉驿等宽正黑&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>本地IP</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="temicon_3">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/tem_ok.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="temp_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>文泉驿等宽正黑</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>33.0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/tem_char.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="humicon_3">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/humi_ok.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="humi_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>文泉驿等宽正黑</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>35.7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/humi_char.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="temicon_2">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/tem_ok.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="temp_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>文泉驿等宽正黑</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>33.0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/tem_char.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="humicon_2">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/humi_ok.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="humi_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>文泉驿等宽正黑</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>35.7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/humi_char.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>65</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="humicon_1">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/humi_ok.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="humi">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>文泉驿等宽正黑</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>35.7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="temp">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>2</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>文泉驿等宽正黑</family>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>33.0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/humi_char.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/tem_char.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="temicon_1">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/png/tem_ok.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ringbtton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/png/beep_ok.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fansbtton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/png/fans_ok.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 30pt &quot;文泉驿等宽正黑&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLabel" name="distance">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 30pt &quot;文泉驿等宽正黑&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
